managed_config/index.bs

```diff
@@ -0,0 +1,98 @@
+<pre class='metadata'>
+Title: Managed Configuration API
+Shortname: managed-configuration
+Level: 1
+Editor: Anatoliy Potapchuk (Google) apotapchuk@chromium.org
+Status: w3c/WD
+Issue Tracking: Github https://github.com/WICG/WebApiDevice/issues
+URL: http://example.com/url-this-spec-will-live-at
+Abstract: This document defines a web platform API that enables developers to access administrator-set configuration on managed devices. Such API invaluable for such use cases as digital signage and device-centric personalization.
+TR: 
+</pre>
+
+<pre class=link-defaults>
+spec:html; type:interface; text:Navigator
+</pre>
+
+<pre class="anchors">
+
+</pre>
+
+# Model # {#model}
+
+## Managed devices ## {#managed-devices}
+
+The API is presumed to be used on devices, which are not fully controlled by the end user, but rather by an external entity, the device administrator. Device administrators are given the power to fully control the managed device.
+
+## Managed configuration ## {#managed-configuration}
+
+For the purposes of this specification the device administrator can control a list of managed web applications. Each entry in this list can have a set JSON configuration, which is accessible by the documents hosted at the origin of the web application.
```

Comment:
It might be nice to explicitly declare the data model as a [map](https://infra.spec.whatwg.org/#maps) where the key is an [origin](https://html.spec.whatwg.org/multipage/origin.html#concept-origin) and the value is a `record<DOMString, object>`.